### DISCUSSION TITLE
Command Menu - what's the "right" place to to perform action on select?

### DISCUSSION THREAD

**Discussion by ItaiYosephi** [ariakit/ariakit#3935] [Questions] [answered]
Following this example:
https://ariakit.org/examples/dialog-combobox-command-menu
Should `ComboboxProvider.setSelectedValue` be used?
Or maybe a prop on the `ComboBoxItem` itself?

**Comment by diegohaz** [comment, marked as answer]
I'd simply pass `onClick` to the item.

If you want to use `setSelectedValue`, you need to pass [`value`](https://ariakit.org/reference/combobox-item#value) to `ComboboxItem`.
